### ISSUE TITLE
Model unable to learn when to stop

### ISSUE THREAD

**Issue by akshitmittal1** [NVIDIA/tacotron2#217]
Hi! I am training on my own data on a pre-trained model. sampling rate of dataset is 22050, batch size is 24, even after 3000 iterations, every time output result is of fixed length. I guess it is not learning when to stop. And the results are just noises.

**Comment by theblackcat102** [comment]
Does the validation loss dropping? I suppose you can keep training until 10k and see if there's any alignment in between the two mel cepstrum outputs. If it doesn't converge probably your data isn't "clean" enough. 
My previous dataset was not properly aligned which causes the model unable to find a good stopping boundary after training for 30k iteration.

**Comment by akshitmittal1** [comment]
No, the validation drop doesn't reduce either.

**Comment by theblackcat102** [comment]
@akshitmittal1  Any screenshots about the tensorboard loss graph?

**Comment by akshitmittal1** [comment]
@theblackcat102 I put the model on training again, this time model perfectly learns when to stop. The thing I changed was reduce the learning rate. The results are good in terms of pronunciation but there are background noises and slight blurryness in the voice. I think that's because of my data. Still, can you suggest me anything which I can do to reduce the background noises
